### DISCUSSION TITLE
Low framerate

### DISCUSSION THREAD

**Discussion by twinflac** [redeclipse/base#1345] [Help and Support] [answered]
Hallo, today im compile your game with sources, after launch it im see the framerate ~2-3, after im set graphics to "very low", noting happend.

 My hardware and os:
   - Manjaro gnu/linux
   - Kernel 5.15.32-1
   - CPU: AMD Phenom 2(II) N979 (4) 2.200GHz
   - GPU: AMD ATI Mobility Radeon HD 5430/5450/5470
   - Memory: 4000mb

P.S. Please help slove this problem.

**Comment by Calinou** [comment, marked as answer]
> GPU: AMD ATI Mobility Radeon HD 5430/5450/5470

This is a very old and low-end laptop GPU, so you probably won't be able to get a playable framerate out of it (even after entering `/screenres 320 200` in chat).

[Red Eclipse 1.6.0](https://github.com/redeclipse/base/releases/tag/v1.6.0) will probably be able to run in a playable manner at low settings, as it uses an older rendering engine.

**Comment by qreeves** [comment]
Not a bug, converting to discussion.
